packages-content-model/roosterjs-content-model-editor/lib/publicApi/utils/formatWithContentModel.ts

```diff
@@ -130,3 +132,19 @@ function handleDeletedEntities(
         }
     );
 }
+
+function handleImages(editor: IContentModelEditor, context: FormatWithContentModelContext) {
+    if (context.images.length > 0) {
+        const viewport = editor.getVisibleViewport();
```

Comment:
Re running getVisibleViewport could have high compute use if used for multiple images, since it runs getBoundingClientRect for the editor div for every image, 
I wonder if we could cache the value if it is used once when formatting, maybe in the context.
@juliaroldi @JiuqingSong do you know if this is a big concern or if caching the value should be a good fix?